RocketApp/app/src/main/java/com/example/rocketapp/IntCountExperiment.java

```diff
@@ -0,0 +1,26 @@
+package com.example.rocketapp;
+import java.util.ArrayList;
+import java.lang.Integer;
+import com.google.firebase.firestore.Exclude;
+
+public class IntCountExperiment extends Experiment {
+
+    private ArrayList<IntCountTrial> trials = new ArrayList<>();
+
+    public IntCountExperiment() {
+        //To be added
```

Comment:
I think if you change this to '//TODO to be added' it makes it easier to track

Reply:
rectified